TCAT/Controllers/AllStopsTableViewController.swift

```diff
@@ -109,7 +104,19 @@ class AllStopsTableViewController: UITableViewController, DZNEmptyDataSetSource,
         }
         
         return sectionIndexDictionary
-
+    }
+    
+    /// Retrieves the keys from the sectionIndexDictionary
```

Comment:
put this function in the table data source section so every function is under a `MARK`-ed section

Reply:
Should I move the other similar function into the table data source section as well?